middleware/http/wasm/httpwasm.go

```diff
@@ -54,7 +58,8 @@ func (m *middleware) getHandler(ctx context.Context, metadata dapr.Metadata) (*r
 			WithRandSource(rand.Reader).
 			WithSysNanosleep().
 			WithSysWalltime().
-			WithSysNanosleep()))
+			WithSysNanosleep()),
+		handler.GuestConfig(wasmConfig))
```

Comment:
can you raise a PR upstream to coerce empty to nil?

Reply:
When writing into memory, the upstream has checked the `len` of config bytes, does it necessary to coerce empty to nil?
```
// getConfig implements the WebAssembly host function handler.FuncGetConfig.
func (m *middleware) getConfig(_ context.Context, mod wazeroapi.Module, stack []uint64) {
	buf := uint32(stack[0])
	bufLimit := handler.BufLimit(stack[1])

	configLen := writeIfUnderLimit(mod.Memory(), buf, bufLimit, m.guestConfig)

	stack[0] = uint64(configLen)
}

func writeIfUnderLimit(mem wazeroapi.Memory, offset, limit handler.BufLimit, v []byte) (vLen uint32) {
	vLen = uint32(len(v))
	if vLen > limit {
		return // caller can retry with a larger limit
	} else if vLen == 0 {
		return // nothing to write
	}
	mem.Write(offset, v)
	return
}
```